src/main/java/TaskList.java

```diff
@@ -0,0 +1,72 @@
+import java.util.ArrayList;
+
+/**
+ * Manages an ArrayList of Tasks.
+ */
+public class TaskList {
+
+    private ArrayList<Task> taskList;
+
+    public TaskList(){
+        this.taskList = new ArrayList<Task>();
+    }
+
+    /**
```

Comment:
I really like that you comment on every method. Makes everything much more readable.

Reply:
Thank you!